Cargo.toml

```diff
@@ -70,3 +75,19 @@ facebook          = ["hyper/ssl", "openssl"]
 portaudio-backend = ["portaudio"]
 pulseaudio-backend= ["libpulse-sys"]
 default           = ["with-syntex", "portaudio-backend"]
+
+[package.metadata.deb]
+maintainer = "nobody"
+copyright = "2015 Paul Lietar"
+license_file = ["LICENSE", "0"]
+depends = "$auto"
```

Comment:
Out of interest, how reliable is `"$auto"` ? What does it use to detect dependencies ?